src/Api/ErrorHandler.php

```diff
@@ -34,8 +35,12 @@ public function __construct(JsonApiErrorHandler $errorHandler)
      * @param Exception $e
      * @return JsonApiResponse
      */
-    public function handle(Exception $e)
+    public function handle(Throwable $e)
     {
+        if (! $e instanceof Exception) {
+            $e = new Exception($e->getMessage(), $e->getCode(), $e);
+        }
```

Comment:
This is only needed until tobscure/json-api supports `Throwable`, right?

Reply:
Yep.